packages/entity-list/src/components/Table/useDnD.js

```diff
@@ -0,0 +1,46 @@
+import {useState} from 'react'
+
+const initialState = {
+  currentlyDragging: null,
+
+  currentlyDragOver: null
+}
```

Comment:
(maybe the blank line could be inserted here:))

Reply:
i moved the blank line :)